source/js/foundation/template-js-handler/article-hero-video-handler.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Adding play/pause functionality to the optional Article Page hero video section.
+ */
+
+ const articleHeroVideoHandler = () => {
+    let articlePageHero = document.querySelector(".article-hero");
```

Comment:
just as a good habit: if the variable is not intended to be reassigned, you'll want to use `const` rather than `let`.

Reply:
Thanks for the catch Pomax, I will remember to do that moving forward! Updated and marking as resolved.